NAS2D/Filesystem.cpp

```diff
@@ -291,34 +292,35 @@ void Filesystem::del(const std::string& filename)
 
 std::string Filesystem::readFile(const std::string& filename) const
 {
-	PHYSFS_file* myFile = PHYSFS_openRead(filename.c_str());
-	if (!myFile)
+	const auto& filePath = findFirstPath(filename, mSearchPaths);
+	if (filePath.empty())
```

Comment:
Use `std::filesystem::exists` here.

```
const auto filePath = std::filesystem::path{findFirstPath(filename, mSearchPaths)}; 
if(!std::filesystem::exists(filePath)) //File doesn't exist
{
//...throw
}
//Guaranteed to exist.

Reply:
Same as previous. That check is already performed internally as part of `findFirstPath`.